app/models/pglogical_subscription.rb

```diff
@@ -1,50 +1,56 @@
 require 'pg/dsn_parser'
 require 'pg/logical_replication'
+require 'active_hash'
 
-class PglogicalSubscription < ActsAsArModel
-  set_columns_hash(
-    :id                   => :string,
-    :status               => :string,
-    :dbname               => :string,
-    :host                 => :string,
-    :user                 => :string,
-    :password             => :string,
-    :port                 => :integer,
-    :provider_region      => :integer,
-    :provider_region_name => :string
-  )
-
-  def self.find(*args)
-    case args.first
-    when :all then find_all
-    when :first, :last then find_one(args.first)
-    else find_id(args.first)
-    end
+class PglogicalSubscription < ActiveHash::Base
+  include Vmdb::Logging
+
+  fields :status, :dbname, :host, :user, :password, :port, :provider_region, :provider_region_name
+
+  # Simple wrapper around the ActiveRecord::Base.connection method.
+  #
+  def self.connection
+    ActiveRecord::Base.connection
   end
 
-  def self.lookup_by_id(to_find)
-    find(to_find)
-  rescue ActiveRecord::RecordNotFound
-    nil
+  # Instance wrapper method around the +connection+ singleton method.
+  #
+  def connection
+    self.class.connection
   end
 
-  singleton_class.send(:alias_method, :find_by_id, :lookup_by_id)
-  Vmdb::Deprecation.deprecate_methods(singleton_class, :find_by_id => :lookup_by_id)
+  # Alias for backwards compatability.
+  class << self
+    alias lookup_by_id find_by_id
```

Comment:
Should we keep the deprecation or remove the method entirely?

Reply:
I'm ok with it either way, was just trying to avoid minimizing the breakage.